Lib/test/test_fcntl.py

```diff
@@ -6,6 +6,7 @@
 import struct
 import sys
 import unittest
+import resource
```

Comment:
@hyeongyun0916  resource module looks like not available on Windows platform.

Reply:
I'll add a function to the test support.